diff --git a/node_modules/@wagmi/core/dist/index.d.ts b/node_modules/@wagmi/core/dist/index.d.ts
index e1e00e2..3c076c6 100644
--- a/node_modules/@wagmi/core/dist/index.d.ts
+++ b/node_modules/@wagmi/core/dist/index.d.ts
@@ -192,7 +192,7 @@ declare function getNetwork(): GetNetworkResult;
 
 type SignMessageArgs = {
     /** Message to sign with wallet */
-    message: string;
+    message: string | { raw: Hex | Uint8Array }
 };
 type SignMessageResult = SignMessageReturnType;
 declare function signMessage(args: SignMessageArgs): Promise<SignMessageResult>;
